portfolio/src/main/webapp/index.html

```diff
@@ -48,6 +62,8 @@ <h2>Some Projects I Worked On</h2>
        <a href="images/proj2.png"><img src="images/proj2.png"/></a>
 </body>
 
+<p id="element-container"></p>
```

Comment:
I might move this closer to the "Get Data" button so you can see the text being updated without having to scroll towards the bottom.

Reply:
The point of this is that the data is being transferred.